forward/ft_compute_leadfield.m

```diff
@@ -356,6 +366,13 @@
         % construct the channels from a linear combination of all magnetometer coils
         lf = sens.tra * lf;
       end
+      
+      % convert lf into siunits: Tm/A
```

Comment:
see previous comments about unit conversions in low-level code

Reply:
deleted unit conversion at this level here: c9080cd